aws-s3/spring-cloud-starter-stream-source-s3/src/main/java/org/springframework/cloud/stream/app/s3/source/AmazonS3SourceProperties.java

```diff
@@ -26,7 +26,7 @@
 /**
  * @author Artem Bilan
  */
-@ConfigurationProperties
+@ConfigurationProperties("s3")
```

Comment:
How this is going to work if we will have `source` and `sink` in the same stream ?